src/server.rs

```diff
@@ -3456,6 +3571,63 @@ errorCounts
         assert_eq!(result, Ok(None));
     }
 
+    // Historically, the completion of a package also brought with it an additional edit
+    // that would import the stdlib module it was referring to. This test asserts that we
+    // don't add it back in, but also serves as documentation that this was a conscious
+    // choice, as the user experience was not good.
+    #[test]
+    async fn test_package_completion_when_it_is_not_imported() {
+        let fluxscript = r#"sql"#;
+        let server = create_server();
+        open_file(&server, fluxscript.to_string()).await;
+
+        let params = lsp::CompletionParams {
+            text_document_position: lsp::TextDocumentPositionParams {
+                text_document: lsp::TextDocumentIdentifier {
+                    uri: lsp::Url::parse(
+                        "file:///home/user/file.flux",
+                    )
+                    .unwrap(),
+                },
+                position: lsp::Position {
+                    line: 0,
+                    character: 2,
+                },
+            },
+            work_done_progress_params: lsp::WorkDoneProgressParams {
+                work_done_token: None,
+            },
+            partial_result_params: lsp::PartialResultParams {
+                partial_result_token: None,
+            },
+            context: Some(lsp::CompletionContext {
+                trigger_kind: lsp::CompletionTriggerKind::INVOKED,
+                trigger_character: None,
+            }),
+        };
+
+        let result =
+            server.completion(params.clone()).await.unwrap().unwrap();
+
+        expect_test::expect![[r#"
+                    {
+                      "isIncomplete": false,
+                      "items": [
+                        {
+                          "label": "sql",
+                          "kind": 9,
+                          "detail": "Package",
+                          "documentation": "sql",
+                          "sortText": "sql",
+                          "filterText": "sql",
+                          "insertText": "sql",
+                          "insertTextFormat": 1
+                        }
+                      ]
+                    }"#]]
+        .assert_eq(&serde_json::to_string_pretty(&result).unwrap());
+    }
```

Comment:
Can you add another test for when there are multiple matches and each match is presented as a choice to the user? 

For example we have repeated package names in the experimental namespace quite often. Like `experimental/array` and `array`. We should offer to import both and let the user decide which code action to take.

Reply:
Yup, test added.